src/main/java/org/mitre/synthea/world/agents/behaviors/providerfinder/ProviderFinderNearest.java

```diff
@@ -19,19 +19,19 @@ public Provider find(List<Provider> providers, Person person, EncounterType serv
     Stream<Provider> options = providers.stream()
         // Find providers that accept the person
         .filter(p -> p.accepts(person, time));
-
+        
     // Find providers with the requested service, if one is given
     if (service != null) {
       options = options.filter(p -> p.hasService(service));
-    }
+    } 
 
     // If it's not an emergency
     if (service == null
```

Comment:
This if statement now does nothing too, so perhaps should be commented out entirely